### ISSUE TITLE
Resources invalid path gives miss leading error

### ISSUE THREAD

**Issue by cowglow** [shakacode/sass-resources-loader#107]
When the file path of the resource is invalid:

```
{
    loader: "sass-resources-loader",
    options: {
        resources: ["./wrong/path/to/file.scss"],
    },
}
```
it spits out the error:
```
Error: 
      Something wrong with provided resources.
      Make sure 'options.resources' is String or Array of Strings.
```

which is misleading because the value of `options.resources` IS a string, but it just invalid

**Comment by toltarisa** [comment]
did you tried **path.resolve(__dirname, './path/to/resource')**

**Comment by cowglow** [comment]
Yes, but that's not the issue.
If the resource path is wrong, the error get thrown. To ensure that it is a string. But it is a string, it's just that the file path is wrong, or maybe the develop misspells it, and so to improve debugging, my PR, aims to improve the error message.

**Comment by justin808** [comment]
I just merged #108.

**Comment by vana-dev** [comment]
same problem...
